operator/cmd/mesh/profile-common.go

```diff
@@ -20,11 +20,10 @@ import (
 	"path/filepath"
 	"strings"
 
+	yaml2 "github.com/ghodss/yaml"
```

Comment:
do we use other yaml package here?

Reply:
no, it doesn't matter, restored the old one.